dts/arm/ti/lm3s6965.dtsi

```diff
@@ -46,6 +46,15 @@
 			status = "disabled";
 			label = "UART_2";
 		};
+
+		eth: ethernet@40048000 {
+			compatible = "ti,stellaris-ethernet";
+			reg = <0x40048000 0x3c>;
+			interrupts = <42 0>;
+			status = "disabled";
+			local-mac-address = [00 11 22 33 44 55];
```

Comment:
If you set some default mac address, then at least use the range allocated for documentation 00-00-5E-00-53-xx, see  RFC 7042 for details.

Reply:
Updated MAC as per your suggestion.